packages/matrix-client-server/src/user/account_data/index.ts

```diff
@@ -0,0 +1,98 @@
+import type MatrixClientServer from '../..'
+import {
+  jsonContent,
+  validateParameters,
+  errMsg,
+  type expressAppHandler,
+  send
+} from '@twake/utils'
+
+interface parameters {
+  userId: string
+  type: string
+}
+
+interface PutRequestBody {
+  content: string
+}
+
+const schema = {
+  content: true
+}
+
+const mxidRe = /^@[0-9a-zA-Z._=-]+:[0-9a-zA-Z.-]+$/
+// TODO : Handle error 403 where the user isn't allowed to see this user's account data
+// TODO : Handle error 405 where the type of account data is controlled by the server and cannot be modified by the client
+
+const accountDataType = (
+  clientServer: MatrixClientServer
+): expressAppHandler => {
+  return (req, res) => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-expect-error
+    const prms: parameters = req.params as parameters
```

Comment:
I'm not a typescript expect but I am surprised the types we defined is identical to variable names.
I would think a good practice is to differenciate types for variable / fields using case. If it starts in capital case then it's a type. Such a convention makes code easier to read.

Bonus: then we do not need to use shortened valiable name to avoid conflicts with the types:

```suggestion
    const paramters: Parameters = req.params as Parameters
```

Also use regular words for variable names. We have IDE auto complete and it avoids misunderstanding / makes code easier to read.

Reply:
Fully agree

Reply:
Will fix this right away too, thank you !